parser/v2/elementparser.go

```diff
@@ -118,19 +120,44 @@ var (
 		attr.NameRange = NewRange(pi.PositionAt(pi.Index()-len(attr.Name)), pi.Position())
 
 		// ="
-		result, ok, err := parse.Or(parse.String(`="`), parse.String(`='`)).Parse(pi)
-		if err != nil || !ok {
+		var index int
+		attributeEquals := []parse.Parser[string]{
```

Comment:
Nit - these can be converted to arrays `[...]` or defined outside of this function to avoid some allocations.

Reply:
I thought the same thing. I'll do a refactor.